### PR TITLE
Remove Universal APK generation

### PR DESCRIPTION
**Pull request by Bence1001**
### Checklist

- [x] I've read and accepted the [Contribution Guidelines](https://github.com/bitrise-steplib/.github/blob/main/CONTRIBUTING.md)
- Requires _MAJOR_ [version update](https://semver.org/)

### Context
* The step can generate a Universal APK for every AAB that doesn't have a corresponding Universal APK already.
* This functionality has been deprecated for a long time and is scheduled for removal (as you can already do that by using the [Export Universal APK step](https://github.com/bitrise-steplib/bitrise-step-export-universal-apk).

### Changes
**Remove `generate_universal_apk_if_none` step input.**
* Remove corresponding logic and tests.

### CODE REVIEW

File: aabutil.go [hunk 1]
```diff
@@ -1,100 +0,0 @@
-package main
```
Comment (Bence1001) [inline aabutil.go:1]: Functions in this file were used from `ensureAABsHasUniversalAPKPair`, which was removed.

File: step.yml [hunk 2]
```diff
@@ -245,14 +234,13 @@ inputs:
         - "false"
         - "true"
   - bundletool_version: "0.13.4"
```
Comment (ofalvai) [inline step.yml:236]: Can't we remove this input now?

**Review by ofalvai — DISMISSED** [review]

File: step.yml (hunk 2 above)
Comment (Bence1001) [reply step.yml:236]: Nope, it's still used for parsing information about the App Bundle. Check https://github.com/bitrise-steplib/steps-deploy-to-bitrise-io/blob/master/uploaders/aabuploader.go#L19.

**Review by mateherber — APPROVED** [review]

**Comment by rusakovic** [comment]
I think this doc should be changed https://bitrise.io/integrations/steps/deploy-to-bitrise-io

that this step is not generated APK anymore

image

**Pull request merged**